agent/bench-scripts/tests/pbench-fio/test-49.sts

```diff
@@ -1 +0,0 @@
-1
```

Comment:
I doubt it; but there are 13 soft links in this directory, 9 of them linking various test `.sts` files to this same `test-14.sts`, not just the three here. I'm going to close my eyes, make funny noises, and pretend I didn't "hear" you point this out because I'm not inclined to rework this apparent legacy "tradition".

Reply:
_Le sigh._